marketplace-service/src/main/java/com/axonivy/market/github/service/impl/GHAxonIvyProductRepoServiceImpl.java

```diff
@@ -263,21 +283,14 @@ public void getExtractedPartsOfReadme(ProductModuleContent productModuleContent,
     } else if (setupIndex != -1) {
       setup = parts[1];
     }
-
-    setDescriptionWithLocale(productModuleContent, description.trim(), locale);
-    productModuleContent.setDemo(demo.trim());
-    productModuleContent.setSetup(setup.trim());
+    locale = StringUtils.isEmpty(locale) ? Language.EN.getValue() : locale.toLowerCase();
```

Comment:
We can use Language.EN.getValue().toLowerCase() instead of using the method toLowerCases() at line 293.